src/libstore/machines.cc

```diff
@@ -56,6 +56,8 @@ ref<Store> Machine::openStore() const {
         storeParams["log-fd"] = "4";
         if (sshKey != "")
             storeParams["ssh-key"] = sshKey;
+        if (sshPublicHostKey != "")
+            storeParams["base64-ssh-public-host-key"] = sshPublicHostKey;
```

Comment:
Maybe both of these should be passed if the storeUri is ssh-ng as well?

Reply:
I think that would make sens indeed (though I guess it would be nice to move towards systematically bundling these directly in the store uri)

Reply:
Done!